webcompat/api/endpoints.py

```diff
@@ -170,22 +169,32 @@ def get_search_results(query_string=None, params=None):
 
 
 @api_bp.route('/issues/<int:number>/comments', methods=['GET', 'POST'])
-@mockable_response
 def proxy_comments(number):
-    """XHR endpoint to get issues comments from GitHub.
+    """XHR endpoint to get issues comments from GitHub, or to add
+    a new comment.
 
     Either as an authed user, or as one of our proxy bots.
     """
     params = request.args.copy()
+    path = 'repos/{0}/{1}/comments'.format(ISSUES_PATH, number)
     if request.method == 'POST' and g.user:
-        path = 'repos/{0}/{1}/comments'.format(ISSUES_PATH, number)
-        return api_request('post', path, params=params,
-                           data=get_comment_data(request.data),
-                           mime_type=JSON_MIME_HTML)
+        new_comment = api_request('post', path, params=params,
+                                  data=get_comment_data(request.data),
+                                  mime_type=JSON_MIME_HTML)
+        return get_html_comments(new_comment)
     else:
-        path = 'repos/{0}/{1}/comments'.format(ISSUES_PATH, number)
-        return api_request('get', path, params=params,
-                           mime_type=JSON_MIME_HTML)
+        # TODO: handle the (rare) case for more than 1 page of comments
```

Comment:
I will file a bug for this, but I think we're fine to deploy without it (since the most comments we have on an issue is 41 right now, well below the 100 this supports).

Reply:
Maybe it could be a question of a load more buttons. or something.